### PR TITLE
Merge branch 'master' into graph-updates & fix revert error

### PR DESCRIPTION
**Pull request by LiorAgnin**
subj

### CODE REVIEW

**Comment by mul53** [comment]
@LiorAgnin also here https://github.com/fuseio/fuse-nft-subgraph/blob/7fa765b6bf6baf37e261f58a5ef61784001df7c6/src/mapping.ts#L61

File: src/mapping.ts [hunk 1]
```diff
@@ -30,9 +30,13 @@ export function handleTransfer(event: Transfer): void {
       item.revealed = false;
       item.tokenId = event.params.tokenId;
       item.collection = collection.id;
-      item.descriptorUri = Erc721.bind(event.address).tokenURI(
+      let tokenURIResult = Erc721.bind(event.address).try_tokenURI(
         event.params.tokenId
       );
+      if (tokenURIResult.reverted) {
```
Comment (leonprou) [inline src/mapping.ts:36]: Can we add some warning message here?

Comment (LiorAgnin) [reply src/mapping.ts:36]: @leonprou Sure, I'll add some.

Comment (leonprou) [reply src/mapping.ts:36]: Thanks LGTM

**Review by mul53 — APPROVED** [review]
Looks Good

**Pull request merged**